packages/schematics/angular/application/other-files/app.module.ts

```diff
@@ -1,4 +1,4 @@
-import { BrowserModule } from '@angular/platform-browser';
+<% if (!experimentalIvy) { %>import { BrowserModule } from '@angular/platform-browser';<% } %>
```

Comment:
I think there is the same problem for `AppRoutingModule`, as at the moment it looks like this is only being used if it's not `experimentalIvy`

https://github.com/angular/angular-cli/blob/151a0b2cec417ad8dc6637dc88f4684cafff3642/packages/schematics/angular/application/other-files/app.module.ts#L11-L13

I think that `AppRoutingModule` shouldn't be effect by the fact if  `experimentalIvy` is enabled or disabled. 

That said, maybe @hansl can confirm about this.

Reply:
You're right. I generated it without the `routing` flag so I didn't see it, but I can add it if @hansl confirms.

Reply:
Yes. I don't even know if we support router in Ivy.

Reply:
I amended the commit to remove the `AppRoutingModule` if the `experimentalIvy` flag is used.